.github/workflows/workflows.yaml

```diff
@@ -0,0 +1,18 @@
+name: Release Helm chart for greeting workflow
```

Comment:
s/greeting/all-workflows/ (or other name of your choice)

Reply:
do you prefer the `workflows aggregator` or `workflows umbrella chart`? Our mutual friend has some advice:
> Both "Workflows Aggregator" and "Workflows Umbrella Chart" convey the idea of a Helm chart that installs dependencies (referred to as workflows in this case). However, the term "Umbrella Chart" is more commonly used in the Helm community to describe charts that manage multiple sub-charts or dependencies.

> Therefore, "Workflows Umbrella Chart" might be the more appropriate and recognizable term for your use case. It clearly indicates that the chart serves as a container for multiple workflows or dependencies, much like an umbrella covering various components. However, you can choose whichever name you feel best suits the purpose and context of your chart.

Reply:
"Workflows Umbrella Chart"  sounds good.